vsts/sdl.yaml

```diff
@@ -21,6 +21,7 @@ phases:
     inputs:
       debugMode: false
       suppressionsFile: vsts/CredScanSuppressions.json
+      toolMajorVersion: V2
```

Comment:
We've been getting warning messages about this as well. v1 is the default and is being deprecated.

Reply:
We are already using v2 in the .NET analyzer scripts, FYI